src/lib/containsEmail.js

```diff
@@ -0,0 +1,15 @@
+import assertString from './util/assertString';
+import isEmail from './isEmail';
+
+export default function containsEmail(str) {
+  assertString(str);
+  let n = str.length;
+  for (let i = 0; i < n; i++) {
+    for (let j = i + 1; j <= n; j++) {
+      if (isEmail(str.substring(i, j))) {
```

Comment:
I suggest you pass an options object to `isEmail` to allow default options customization.